uxarray/core/dataset.py

```diff
@@ -277,6 +279,13 @@ def integrate(self, quadrature_rule="triangular", order=4):
         # Compute the integral
         >>> integral = uxds.integrate()
         """
+
+        # TODO: Deprecation Warning
+        warn(
+            "Integration of a single data variable will be deprecated in a future release. For integrating"
+            "a single variable, use the UxDataArray.integrate() method.",
+            DeprecationWarning)
+
```

Comment:
Please replace the text here with something like: 

"This method currently works only if there is a single data array in the dataset. For integration of a single data variable, use the UxDataArray.integrate() method instead. This will be deprecated, i.e. replaced with a version that can work with multiple data arrays in a dataset in a future release."